superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -571,6 +571,16 @@ export const controls = {
     }),
   },
 
+  polygon: {
+    type: 'SelectControl',
+    label: t('Polygon Column'),
+    validators: [v.nonEmpty],
+    description: t('Select the polygon column'),
```

Comment:
Looks like we're assuming JSON formatted string here based on the code bellow. We should be clear about that here. The column the user points to should contain JSON array(N) of array(2).

Reply:
I wonder if we ever use polyline for polygons, seems like it should work for both. We may want to support both as we do for `deck_path` (optional). That's only immediately useful if we have that input format internally.